Backend/router/classroomroute.js

```diff
@@ -2,10 +2,9 @@ const express = require("express");
 const classroomcontroller = require("../controllers/classroomc");
 const router = express.Router();
 // show all data
-router.post("/create", classroomcontroller.createClassroom);
-router.get("/classroom", classroomcontroller.getAllClassrooms);
+// router.post("/create", classroomcontroller.createClassroom);
+router.get("/allclassroom", classroomcontroller.getAllClassrooms);
```

Comment:
An appropriate endpoint name would have been `classrooms`, as per the conventions.